web/libs/js/main.dash2.js

```diff
@@ -2595,7 +2595,7 @@ $('body')
             $.confirm.e.modal('show');
             $.confirm.title.text('<%-cleanLang(lang['Delete Video'])%> : '+e.file)
             e.html='<%-cleanLang(lang.DeleteVideoMsg)%>'
-            e.html+='<video class="video_video" autoplay loop controls><source src="'+e.href+'" type="video/'+e.mon.ext+'"></video>';
+            e.html+='<video class="video_video" autoplay loop controls><source src="'+e.href+'" type="video/'+e.mon.ext+'"><track default kind="subtitles" src="' + e.href.replace('/videos/','/subtitles/').replace('.'+e.mon.ext,'.'+e.mon.ext+'.vtt') + '"></video>';
```

Comment:
`subtitleHref` or `timestamp` maybe to avoid confusion. I was planning to allow adding dynamic text in the future.